### PR TITLE
Fix #5997: Last co-ordinates for the javax. namespace for version 9

### LINKED ISSUE #5997: General : Please use the last javax. maven coordinates for Version 9
**Description**
Please use the final and last jars (Not version 3) for the namespace

**Describe the solution you would like**
Please implement and use the last jars to be released for the javax. namespace

**Additional context**
This must not/should not conflict with any existing source code base in any way

### PR DESCRIPTION
**Pull request by GedMarc**

### CODE REVIEW

**Comment by tandraschko** [comment]
why do we need jaxb?

**Comment by tandraschko** [comment]
also we should never add a impl dependency to PF

File: pom.xml [hunk 1]
```diff
@@ -142,10 +151,24 @@
...
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>2.3.3</version>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.3</version>
+            <optional>true</optional>
```
Comment (melloware) [inline pom.xml]: Why do we need this runtime?  PF should not need this

File: pom.xml [hunk 2]
```diff
@@ -142,10 +151,24 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.3.3</version>
+            <optional>true</optional>
```
Comment (melloware) [inline pom.xml]: Why do we need this impl PF should not need this...only the API.

File: pom.xml [hunk 3]
```diff
@@ -97,13 +98,21 @@
...
-            <scope>provided</scope>
+            <groupId>jakarta.el</groupId>
+            <artifactId>jakarta.el-api</artifactId>
+            <version>3.0.3</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>jakarta.el</artifactId>
+            <version>3.0.3</version>
```
Comment (melloware) [inline pom.xml]: Impl not needed...

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by GedMarc** [comment]
Hmm apparently for the tests
Let's see how scoping them into tests will work

**Comment by melloware** [comment]
Well it looks like ValueExpressionAnaylyzer test is failing with an initialization failure.

**Comment by melloware** [comment]
@GedMarc @tandraschko I think the reason he added the JAXB-impl was for the unit tests because in Java 11+ they no longer include JAXB.  Same problem we had with the Showcase we had to update if you want to run JDK11.  It works fine without the impl in Java 8.  But we should probably add it at "TEST" scope so the PF build works in JDK8 or 11.  

Also I wonder if we should have our Travis build be Oracle JDK8 and OpenJDK 11 as the two JDK's we build for?

**Comment by tandraschko** [comment]
its ok for me as test dependency, otherwise -1

+1 to add a java11 travis build

**Comment by melloware** [comment]
OK I will work on adding JDK11 to Travis.

**Comment by melloware** [comment]
PR https://github.com/primefaces/primefaces/pull/6013 supersedes this PR so closing.

**Pull request closed without merging**